specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json

```diff
@@ -433,6 +433,119 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareCloudSimple/locations/{regionId}/privateClouds/{pcName}/customizationPolicies": {
+      "get": {
+        "description": "Returns list of customization policies in region for private cloud",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "customizationPolicies"
+        ],
+        "summary": "Implements get of customization policies list",
+        "operationId": "customizationPolicies_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/regionId"
+          },
+          {
+            "$ref": "#/parameters/pcName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "enum": [
+              "linux",
+              "windows",
+              "other"
+            ],
+            "type": "string",
+            "description": "If specified response will be filtered by guest os type",
```

Comment:
Per Microsoft REST API guidelines, this should follow odata $filter syntax: https://github.com/Microsoft/api-guidelines/blob/master/Guidelines.md#97-filtering

Here's a relevant example: https://docs.microsoft.com/en-us/rest/api/batchservice/odata-filters-in-batch#list-support-images